scrapers/MyCherryCrush.yml

```diff
@@ -0,0 +1,59 @@
+name: MyCherryCrush
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://mycherrycrush.com/
+    scraper: sceneScraper
+
+sceneByName:
+  action: scrapeXPath
+  scraper: sceneSearch
+  queryURL: "https://mycherrycrush.com/members/blog/?s={}"
+
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class='entry-title']
+      URL: //link[@rel="canonical"]/@href
+      Date:
+        selector: //span[@class='entry-date']
+        postProcess:
+          - parseDate: January 2, 2006
+      Tags: 
+        Name: //span[@class='entry-category']/a
+      Performers:
+        Name:
+          fixed: Cherry Crush
+      Studio:
+        Name:
+          fixed: MyCherryCrush.com

```

Comment:
The studio should be `MyCherryCrush`

Reply:
> The studio should be `MyCherryCrush`

Shoot, well in StashDB it is MyCherryCrush.com, should I still change it?

Reply:
With studio aliases available in stash it doesnt really matter as any user can create an alias to match the scraper
The same thing for stashdb matches as the first time it is matched from stashdb it can be linked from there.
@sevenfifteen  any name is fine, if stashdb already has the studio just use that name